eZ/Publish/Core/Repository/RoleService.php

```diff
@@ -946,6 +1000,19 @@ public function newRoleCreateStruct(string $name): APIRoleCreateStruct
         );
     }
 
+    /**
+     * Instantiates a role copy class.
+     */
```

Comment:
CS: no need to repeat that from API.
```suggestion
    /**
     * Instantiates a role copy class.
     */
```